cmd/maya-exporter/app/collector/zvol/metrics.go

```diff
@@ -437,6 +438,17 @@ func (p *poolSyncMetrics) withZpoolStateUnknown() *poolSyncMetrics {
 	return p
 }
 
+func (l *poolSyncMetrics) withRequestRejectCounter() *poolSyncMetrics {
+	l.zpoolListRequestRejectCounter = prometheus.NewGauge(
+		prometheus.GaugeOpts{
+			Namespace: "openebs",
+			Name:      "zpool_list_request_reject_count",
+			Help:      "Total no of rejected requests of zpool list",
```

Comment:
Name should be actual command separated by underscore

Reply:
Done. PTAL